counterpartylib/test/fixtures/vectors.py

```diff
@@ -1472,8 +1472,8 @@
             'in': (ADDR[0], None, 'NOSATOSHI', 1000.5, True, False, None, None, '', None, None, DP['default_block_index']),
             'out': (0, 0.0, ['quantity must be in satoshis'], 0, '', True, None, None)
         }, {
-            'in': (ADDR[0], None, 'CALLPRICEFLOAT', 1000, True, False, None, 100.0, '', None, None, DP['default_block_index']),
-            'out': (0, 0.0, [], 0, '', True, False, None)
+            'in': (ADDR[0], None, 'CALLPRICEFLT', 1000, True, False, None, 100.0, '', None, None, DP['default_block_index']), # CALLPRICE AS FLOAT
+            'out': (0, 0.0, [], 50000000, '', True, False, None)
```

Comment:
I wonder how these didn't trigger errors in the test suite until now.  I guess there is no validation on the length in this part of the code.